pkg/ebpf/tracee.go

```diff
@@ -1484,31 +1503,22 @@ func (t *Tracee) Run(ctx gocontext.Context) error {
 
 // Close cleans up created resources
 func (t *Tracee) Close() {
-	err := capabilities.GetInstance().Required(
```

Comment:
Doesn't this function require capabilities?

Reply:
I had no errors when detaching and/or closing the resources, have you ? The cgroup will raise particular privilege for mounts and umounts. The others are mostly Close() I believe.